Makefile

```
@@ -1,3 +1,7 @@
+DEFAULT_GOARCH := $(shell go env GOARCH)
+BUILD_HASH = $(shell git rev-parse HEAD)
```

It might make it easier if we use the short hash?
```suggestion
BUILD_HASH = $(shell git rev-parse --short HEAD)
```

Also, we could populate the version using the tags (which makes it easy to use the matterbuild plugin to set off a new version). Just a thought for the future, something like:
```
BUILD_TAG_LATEST = $(shell git describe --tags --match 'v*' --abbrev=0)
```

---

I was mainly being consistent with both MM and calls plugin as they both use the extended hash. Not a big deal though, we can revisit this if needed.

> Also, we could populate the version using the tags (which makes it easy to use the matterbuild plugin to set off a new version). 

Absolutely.